src/github/pullRequestOverview.ts

```diff
@@ -525,6 +527,16 @@ export class PullRequestOverviewPanel extends IssueOverviewPanel<PullRequestMode
 		}
 	}
 
+	private async openCommitChanges(message: IRequestMessage<{ commitSha: string }>): Promise<void> {
```

Comment:
Add a type fo `{ commitSha: string }` to common/views.ts. Use the new type here and where the `pr.openCommitChanges` is posted from the webview